docker-compose.yml

```diff
@@ -1,23 +1,52 @@
 version: '2'
 services:
-  backend:
-    image: debian:jessie
-    volumes:
-      - .:/var/www
   nginx:
     build: docker/nginx
+    links:
+      - php
     ports:
-      - 80
+      - ${IP_ADDRESS}:80:80
     volumes_from:
-      - backend
+      - php
   php:
     build: docker/php
-    volumes_from:
-      - backend
+    links:
+      - db:db.${PROJECT}.dev
+      - mailhog
+      - redis
```

Comment:
I thought we would focus on PhpMyAdmin and Mailhog. No SOLR or anything else yet